sonar-application/src/test/java/org/sonar/application/AppLoggingTest.java

```diff
@@ -244,6 +247,27 @@ public void fail_with_IAE_if_app_property_unsupported_level() {
     underTest.configure(props);
   }
 
+  @Test
+  public void no_info_log_from_hazelcast() throws IOException {
+    props.set(ClusterProperties.Parameter.ENABLED.getName(), "true");
+    new ClusterProperties(props).populateProps(props);
+    underTest.configure(props);
+
+    assertThat(
+      LoggerFactory.getLogger("com.hazelcast").isInfoEnabled()
+    ).isEqualTo(false);
+  }
+
+  @Test
+  public void configure_logging_for_hazelcast() throws IOException {
+    props.set(ClusterProperties.Parameter.ENABLED.getName(), "true");
+    props.set(ClusterProperties.Parameter.LOG_LEVEL.getName(), "INFO");
+    underTest.configure(props);
+
+    assertThat(
+      LoggerFactory.getLogger("com.hazelcast").isInfoEnabled()
+    ).isEqualTo(true);
+  }
   private void emulateRunFromSonarScript() {
```

Comment:
missing line return

did you reformat the code with the Eclipse Formatter ? (CTRL+ALT+L in Idea)